### PR TITLE
ui-speedspacechart: fix non sticky unit icon on right axis

### LINKED ISSUE #385: GEV: ‰ unit not sticky on top of right ticks
### What happened?

the ‰ unit follows the height scale when we use the enlarger button
https://github.com/user-attachments/assets/a295824b-1c45-46d3-bd5f-8429ae4bffcd

### What did you expect to happen?

it should be sticky on top-right of the right-ticks layer, and follow the axact representation of the mockup
Capture d’écran 2024-08-02 à 10 39 14

### LINKED ISSUE #8312: GEV: ‰ unit not sticky on top of right ticks
> [!NOTE]
> ui-core related [issue](https://github.com/OpenRailAssociation/osrd-ui/issues/385).

### What happened?

the ‰ unit follows the height scale when we use the enlarger button
https://github.com/user-attachments/assets/a295824b-1c45-46d3-bd5f-8429ae4bffcd

### What did you expect to happen?

it should be sticky on top-right of the right-ticks layer, and follow the axact representation of the mockup
Capture d’écran 2024-08-02 à 10 39 14

### How can we reproduce it (as minimally and precisely as possible)?

1. create a scenario with trains
2. enable declivities
3. see the ‰ while enlarging the GEV

### On which environments the bug occurs?

Dev (DgexSol)

### On which browser the bug occurs?

Firefox

### OSRD version (top right corner `Account` button > `Informations`)

dev 34c0b8e

### PR DESCRIPTION
**Pull request by RomainValls**
closes #385
closes https://github.com/OpenRailAssociation/osrd/issues/8312

### CODE REVIEW

**Review by Caracol3 — COMMENTED** [review]
LGTM but can't be tested in storybook

File: ui-speedspacechart/src/components/const.ts [hunk 1]
```diff
@@ -4,6 +4,13 @@ export const SLOPE_FILL_COLOR = '#CFDDCE';
 
 export const RIGHT_TICK_HEIGHT_OFFSET = 2;
 
+export const TICK_ICONS = {
```
Comment (clarani) [inline ui-speedspacechart/src/components/const.ts]: I think `TICK_TITLE_MARGINS` would be a better name, what do you think ?
```suggestion
export const TICK_TITLE_MARGINS = {
```

**Review by clarani — APPROVED** [review]
LGTM, tested in osrd :white_check_mark: 

Feel free to resolve the comment once you're done

**Review by achrafmohye — APPROVED** [review]
LGTM

File: ui-speedspacechart/src/components/const.ts (hunk 1 above)
Comment (RomainValls) [reply ui-speedspacechart/src/components/const.ts]: You're right, done 👍

**Pull request merged**